Automattic/woocommerce-payments
# Improve the transition from the WCPay KYC to the WC Admin Payments Task

Fixes https://github.com/woocommerce/woocommerce/issues/37542

#### Changes proposed in this Pull Request



Improve the transition from the WCPay KYC to the WC Admin Payments Task.



When a user from the payment task completes the KYC, send users back to the payment overview page with the success message.

Code changes:

- Remove the `WCADMIN_PAYMENT_TASK` case from the switch statement in the `get_onboarding_return_url` method 







#### Testing instructions





* [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&woocommerce-payments-dev-tools&branches.woocommerce-payments=update/redirect-to-payment-screen-for-wcadmin-task) 🚀
* Go to WooCommerce
* Select the US as store country
* Go to WooCommerce > Home
* Click on `payment task`
* Click on `Connect your store` 
* Complete KYC https://woocommerce.com/document/woopayments/testing-and-troubleshooting/testing/
* Confirm you're redirected back to payment overview screen with success message

<img width="1351" alt="Screenshot 2024-05-31 at 12 26 41" src="https://github.com/Automattic/woocommerce-payments/assets/4344253/a149f9f9-35c6-4c55-8703-0a79cd62d114">



-------------------


**Post merge**
